src/channel.cpp

```diff
@@ -237,31 +238,23 @@ void Channel::remMod(string& dest) {
 }
 
 bool Channel::isMod(ConnectionPtr con) {
-    if (con->globalModerator || con->admin || (owner == con->characterName) || moderators.find(con->characterName) != moderators.end())
-        return true;
+    return (owner == con->characterName) || moderators.count(con->characterName) > 0;
```

Comment:
Is count faster than find? Seems like it would be the other way around. (not the only instance)

Reply:
They are roughly the same time for any container that is associative, as there can be a maximum of one result returned and it is optimized internally for that case. It however skips dealing with iterators and comparing them to the result of a function call from our side. Shorter logic, easier to read.